### DISCUSSION TITLE
support for Zotero 7

### DISCUSSION THREAD

**Discussion by feeuoo** [tankwyn/WPS-Zotero#25] [General]
I recently updated to Zotero 7. WPS-Zotero doesn't support it. 
![image](https://github.com/tankwyn/WPS-Zotero/assets/53679035/be18736b-11cf-4383-be56-677e5dbcfb51)

**Comment by tankwyn** [comment]
It's an issue with WPS on Windows. The latest WPS version forbids running local program using JS which disables the auto-start of proxy.py. As a workaround, you'll have to  run proxy.py manually before using this addon.

**Reply by fluman4u** [reply]
WPS-Zotero can not run with zotero 7.08  and wps 12.1.0.17885 on ubuntu 24.04LTS.
![截图 2024-10-23 11-07-54](https://github.com/user-attachments/assets/4ccc693d-0fe6-45ea-8a51-cc34a0ee9d0a)
I am sure Zotero is running well.

**Comment by yu547** [comment]
你好，在使用过程中出现这个报错，WPS-Zotero: Client error 400。卸载重装插件后也继续重新提示，求教怎么解决这个问题
